lib/versions.js

```diff
@@ -8,9 +8,19 @@ module.exports.getVersionInfo = lastRelease => {
     return undefined
   }
 
-  return {
+  let patchVersions = lastRelease.patchVersions || true
+
+  let versionInfo = {
     incrementedMajor: semver.inc(lastVersion, 'major', true),
     incrementedMinor: semver.inc(lastVersion, 'minor', true),
     incrementedPatch: semver.inc(lastVersion, 'patch', true)
   }
+
+  if (patchVersions == false) {
+    Object.keys(versionInfo).forEach(function(key) {
+      versionInfo[key] = versionInfo[key].replace(/\.\d+$/, '')
```

Comment:
regex seemed easier.
Should we delete the incrementedPatch key when `patchVersions` is false to make it obvious that the user has to use `NEXT_MINOR_VERSION` or `NEXT_MAJOR_VERSION`?